controllers/locations/_states_list_toolbar.htm

```diff
@@ -6,7 +6,7 @@
         data-size="huge" 
         href="javascript:;" 
         class="btn btn-sm btn-secondary oc-icon-file">
-        Create State
+        <?= e(trans('backend::lang.form.create')) ?> <?= e(trans('rainlab.location::lang.state.label')) ?>          
```

Comment:
```suggestion
        <?= e(trans('backend::lang.form.create_name', ['name' => trans('rainlab.location::lang.state.label')])) ?>
```

Reply:
backend::lang.form.create_name does not exist.

I replaced it with:

`<?= e(trans('backend::lang.form.create_title', ['name' => e(trans('rainlab.location::lang.state.label'))])) ?>`

This works!